cmds/create_translations.go

```diff
@@ -45,7 +45,12 @@ type GoogleTranslateTranslation struct {
 }
 
 func NewCreateTranslations(options common.Options) *createTranslations {
-	languages := common.ParseStringList(options.LanguagesFlag, ",")
+	var languages []string
+	if options.LanguagesFlag == "" {
+		languages = options.LanguagesArrayFlag
+	} else {
+		languages = common.ParseStringList(options.LanguagesFlag, ",")
+	}
```

Comment:
**Callout:** Since Cobra handles parsing strings as arrays we will only need to do conversion of the legacy implemenation.